packages/client/src/pages/RecommendationsPage/RecommendationsTable/RecommendationsTable.tsx

```diff
@@ -209,9 +216,11 @@ const RecommendationsTable: FunctionComponent<RecommendationsTableProps> = ({
               }}
               onRowClick={handleRowClick}
               disableColumnFilter
-              pagination
               pageSize={pageState.pageSize}
-              rowsPerPageOptions={[25, 50, 100]}
+              components={{
+                Toolbar: CustomPagination(handlePageSizeChange),
+                Pagination: CustomPagination(handlePageSizeChange),
```

Comment:
To fix the double return and the type issue in the Pagination file, the assigned components can be changed to return ReactElements as followed:

```suggestion
                Toolbar: () => CustomPagination(handlePageSizeChange),
                Pagination: () => CustomPagination(handlePageSizeChange),
```

Reply:
ok